backend/logs_helper/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from .views import LogsHelperView
+
+logs_helper_get = LogsHelperView.as_view({"get": "get_logs", "post": "store_log"})
```

Comment:
```suggestion
logs_helper = LogsHelperView.as_view({"get": "get_logs", "post": "store_log"})
```

Reply:
No, the persisted logs will not be visible to the user once they log out and log back in. This is because the logs are tied to the session ID, which changes with each new login session.

Reply:
ok, It would be good if we could clear the persisted logs on logout to save the space.

Reply:
Now persisted logs are cleared on logout. Thank you for pointing out.